src/core/model_config.proto

```diff
@@ -661,6 +730,22 @@ message ModelDynamicBatching
   //@@     order.
   //@@
   bool preserve_ordering = 3;
+
```

Comment:
We need to talk some more about the implementation. I think we will need to have the config specify the number of priority levels (since we will need a queue for each we will encourage the use of as few as possible). And then for each there can be a timeout policy. The default will handle the case where priority is disabled and for priority levels where there is no specific policy given.

Reply:
I agree. But I don't think we need a separate field to specify the number of priority levels, if the user wants to specify the priority level as they need, they just specify it in the map below. The empty map simply means priority handling is disabled, as all request will fallback to default policy.

Reply:
That works but introduces a couple of cases where the user has to do something they don't want to. For example, what if I want 3 priority levels (0, 1, 2) that all use a default policy. I need to have a default policy but also a policy for priority level 2 so that I can add 2 to the map with that policy (I assume that we determine the total number of priority levels from the highest value in the map).

Similarly user always has to have a non-default policy for the highest priority level since that needs to go in the map, even if that policy is the same as the default.